samples/TodosApi.Dapper/TodosApi.cs

```diff
@@ -52,7 +52,7 @@ await db.QuerySingleOrDefaultAsync<Todo>("SELECT * FROM Todos WHERE Id = @id", n
         if (!inputTodo.IsValid)
             return Results.Extensions.ValidationProblem(inputTodo.Errors);
 
-        return await db.ExecuteAsync("UPDATE Todos SET Title = @Title, IsComplete = @IsComplete WHERE Id = @Id", inputTodo.Value) == 1
+        return await db.ExecuteAsync($"UPDATE Todos SET Title = @Title, IsComplete = @IsComplete WHERE Id = {id}", inputTodo.Value) == 1
```

Comment:
Rather than using SQL injection let's just set the `inputTodo.Value.Id` to the value of the `id` parameter:
```suggestion
        inputTodo.Value.Id = id;
        return await db.ExecuteAsync("UPDATE Todos SET Title = @Title, IsComplete = @IsComplete WHERE Id = @Id", inputTodo.Value) == 1
```